src/app/shared/footer/footer.component.scss

```diff
@@ -0,0 +1,57 @@
+.footer-container {
+  width: 100%;
+  height: auto;
+  padding: 15px 0;
+  background: url(/assets/background/blogbg.png);
+  .nav-links {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    .create,
+    .follow,
+    .support {
+      .title {
+        position: relative;
+        font-family: "Open Sans";
```

Comment:
asta ar trebui scris asa

font-family: Open-sans;

de asta nu iti arata ca in design

"Open Sans" nu e la fel ca Open-sans - asta pentru ca noi l-am definit intr-un font face